controllers/scopeinstance_controller.go

```diff
@@ -208,7 +207,7 @@ func (r *ScopeInstanceReconciler) ensureBindings(ctx context.Context, in *operat
 			if util.IsOwnedByLabel(existingCRB.DeepCopy(), in) &&
 				reflect.DeepEqual(existingCRB.Subjects, crb.Subjects) &&
 				reflect.DeepEqual(existingCRB.Labels, crb.Labels) {
-				r.logger.Info("Existing ClusterRoleBinding does not need to be updated")
+				log.Log.Info("Existing ClusterRoleBinding does not need to be updated")
```

Comment:
Is there any specific reason for this change? I am not sure about the difference in these log changes.

Reply:
I think this change is fine so that it is more consistent with all our other log calls.

One nit for the actual log message itself, we should probably avoid captialization unless it is a proper noun like `ClusterRoleBinding`:
```suggestion
				log.Log.Info("existing ClusterRoleBinding does not need to be updated")
```

Reply:
@laxmikantbpandhare I looked at main.go and we do not setup the logger on the reconcilers in there. https://github.com/awgreene/oria-operator/blob/main/main.go#L92-L105

We create an instance of ScopeInstanceReconciler that means the logger *logrus.Logger will be nil in this struct https://github.com/awgreene/oria-operator/blob/main/controllers/scopeinstance_controller.go#L44-L49

I'm not sure why we need 2 loggers.  Here we use the log from controller-runtime https://github.com/awgreene/oria-operator/blob/main/controllers/scopeinstance_controller.go#L150 then here we attempt to use the logger from the ScopeInstance https://github.com/awgreene/oria-operator/blob/main/controllers/scopeinstance_controller.go#L211

So the reason for changing this is 1) to prevent the nil pointer error and 2) we seem to be using two different loggers.

Reply:
@everettraven I can fix the capitalization